# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/MediaTransportParent.h
# Commit: 8196045d38b5
# Full Hash: 8196045d38b50e04313725215c36ba1618c50486
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Create an ipdl struct called WebrtcProxyConfig, to cut down on the arg count on lots of functions. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45562
# ==============================================================================

diff -r 6e1eba5c88ad -r 8196045d38b5 dom/media/webrtc/MediaTransportParent.h
--- a/dom/media/webrtc/MediaTransportParent.h	Wed Sep 18 14:27:42 2019 +0000
+++ b/dom/media/webrtc/MediaTransportParent.h	Tue Sep 17 18:19:04 2019 +0000
@@ -24,10 +24,8 @@
   mozilla::ipc::IPCResult RecvCreateIceCtx(
       const string& name, nsTArray<RTCIceServer>&& iceServers,
       const RTCIceTransportPolicy& icePolicy);
-  mozilla::ipc::IPCResult RecvSetProxyConfig(const dom::TabId& tabId,
-                                             const net::LoadInfoArgs& args,
-                                             const nsCString& alpn,
-                                             const int& proxyPolicy);
+  mozilla::ipc::IPCResult RecvSetProxyConfig(
+      const net::WebrtcProxyConfig& aProxyConfig);
   mozilla::ipc::IPCResult RecvEnsureProvisionalTransport(
       const string& transportId, const string& localUfrag,
       const string& localPwd, const int& componentCount);